lib/stormpath-sdk/jwt/authentication_result.rb

```diff
@@ -0,0 +1,7 @@
+module Stormpath
+  module Jwt
```

Comment:
Not sure why we should have a "Jwt" module at all because "Jwt" is what we use to exchange information with the server, not something that indicates any type of functionality. It's almost like having a "Json" module; we use Json to communicate with the server, but that's not something we should expose as a module.

In other SDKs I see all of the token management related classes under the Oauth package (the equivalent for module in Ruby).

Let me know if I'm missing something.